roles/openshift_health_checker/library/docker_info.py

```diff
@@ -0,0 +1,50 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about the docker host.
+"""
+
+from ansible.module_utils.docker_common import *
+
+
+class DockerInfo(DockerBaseClass):
+
+    def __init__(self, client, results):
+
+        super(DockerInfo, self).__init__()
+
+        self.client = client
+        self.results = results
+
+        self.results['info'] = self.get_info()
+
+    def fail(self, msg):
+        self.client.fail(msg)
+
+    def get_info(self):
+        """
+        Lookup and inspect information about the docker host.
+        :returns dictionary of host info.
+        """
+
+        return self.client.info()
+
+
+def main():
+    argument_spec = dict(
+        name=dict(type='list'),
+        )
+
+    client = AnsibleDockerClient(
+        argument_spec=argument_spec
+    )
+
+    results = dict(
+        changed=False,
+        info=[]
+    )
+
+    DockerInfo(client, results)
```

Comment:
Got it, makes sense... this is to get Docker config like configured registries. My bad, took me some time to fully understand it.